spring-cloud-previews/google-cloud-dataproc-metastore-spring-starter/src/main/java/com/google/cloud/metastore/v1/spring/DataprocMetastoreFederationSpringAutoConfiguration.java

```diff
@@ -171,6 +171,33 @@ public DataprocMetastoreFederationSettings dataprocMetastoreFederationSettings(
               clientSettingsBuilder.getFederationSettings().getRetrySettings(), serviceRetry);
       clientSettingsBuilder.getFederationSettings().setRetrySettings(getFederationRetrySettings);
 
+      RetrySettings listLocationsRetrySettings =
```

Comment:
Corresponds to upstream changes in https://github.com/googleapis/google-cloud-java/pull/9350